### PR TITLE
chore: refactor input loop in `get_exec_command_args`

### PR DESCRIPTION
**Pull request by guilhem-barthes**
## Description

Refactor 3 loops on inputs in `substrapp` in only one

## How has this been tested?

CI

## Checklist

- [x] [changelog](../CHANGELOG.md) was updated with notable changes
- [ ] documentation was updated

### CODE REVIEW

**Review by oleobal — APPROVED** [review]
nice

File: backend/substrapp/compute_tasks/command.py [hunk 1]
```diff
@@ -49,40 +50,30 @@ def get_exec_command_args(ctx: Context) -> list[str]:
...
-                id=input.identifier,
-                value=os.path.join(in_models_dir, input.model.key),
-                multiple=ctx.function.inputs[input.identifier].multiple,
+    for input_asset in ctx.input_assets:
+        identifier = input_asset.identifier
+        multiple = ctx.function.inputs[input_asset.identifier].multiple
+
+        value = None
+        # `cast()` are needed as we use `Optional`
+        if input_asset.kind == orchestrator.AssetKind.ASSET_MODEL:
```
Comment (ThibaultFy) [inline backend/substrapp/compute_tasks/command.py:59]: add `and input_asset.model is not None` and remove cast ?

Comment (ThibaultFy) [reply backend/substrapp/compute_tasks/command.py:59]: same under

**Review by ThibaultFy — APPROVED** [review]
lgtm ! :)

**Comment by Owlfred** [comment]
End to end tests: :hourglass_flowing_sand: **STARTING**

**Comment by guilhem-barthes** [comment]
/e2e --tests sdk,frontend,substrafl --benchmarks mnist,camelyon

**Comment by Owlfred** [comment]
End to end tests: :hourglass_flowing_sand: **STARTING**

**Pull request merged**